addons/calendar/static/src/js/calendar_view.js

```diff
@@ -50,11 +52,39 @@ var AttendeeCalendarPopover = CalendarPopover.extend({
             self.event.record.attendee_status = selectedStatus;  // FIXEME: Maybe we have to reload view
             self.$('.o-calendar-attendee-status-text').text(self.statusInfo[selectedStatus].text);
             self.$('.o-calendar-attendee-status-icon').removeClass(_.values(self.statusColors).join(' ')).addClass(self.statusInfo[selectedStatus].color);
+
+            // Update grid event style
+            var record_id = self.event.record.id;
```

Comment:
no it doesn't work I guess that because it's reloading the popup template but not the calendar one, I can't access to the calendar AttendeeCalendarRenderer inside the function

Reply:
So I played around with this locally and here is what I came up with:

You can trigger_up a new 'render_events' custom event.
Then define a handler for that event in the base calendar_renderer.js that just calls '_renderEvents'.

I tested it and it seems to work.

Again, make a separate commit for this, we will ask JS team validation.